### DISCUSSION TITLE
Windows 11

### DISCUSSION THREAD

**Discussion by Switch0r** [NiyaShy/XB1ControllerBatteryIndicator#80] [General]
Is this working on Windows 11 and is this showing the % of the battery?

![Schermafbeelding 2023-07-31 190435](https://github.com/NiyaShy/XB1ControllerBatteryIndicator/assets/1339447/4f5040f1-cc60-47f7-b4ee-3d48f9b5067e)

**Comment by NiyaShy** [comment]
Hey,
can't say for sure if it works on Win11 because I will boycott it as long as I can. But it should, if it wouldn't I'd probably have gotten several reports/complaints. General caveat is that you pretty much have to use one of the proprietary dongles and not Bluetooth, otherwise you'll just get random or no status at all.
And about percentage, the XInput API can't report percentage values, just vague levels (full/medium/low/empty). If you get a % value, you either have the controller connected via BT, or Microsoft did something proprietary with Win11 (again...) that can get more info than the official API.
